react-native/react/menubar/index.render.js.flow

```diff
@@ -9,8 +9,14 @@ export default class Render extends Component {
 }
 
 export type FolderInfo = {
-  folderName: string,
-  isPublic: boolean,
-  isEmpty: boolean,
-  openFolder: () => void
+    type: 'folder',
+    folderName: string,
+    isPublic: boolean,
+    isEmpty: boolean,
+    openFolder: () => void
+} | {
+    type: 'entry',
+    prefix: string,
+    isPublic: boolean,
+    openFolder: (val: string) => void
```

Comment:
I don't know how I feel about this. We have a type called FolderInfo because it's information about a folder. The entry isn't about a folder, it's just data to help render an input to search folders, right?

I think `FolderInfo` should just be it's own thing and have a separate type for `FolderEntry`. I don't think you're gaining much by having this a disjoint union and it's a bit confusing whether we're dealing with a list of data about folders or if there is some entry data in there too.

Also what does `isPublic` for an `entry` mean?